docs/pages/project/contributing/contributing-docs.md

```diff
@@ -316,25 +316,27 @@ Just for fun, here are all the types:
 {% include alert.html type="primary" content="This is a primary alert" %}
 {% include alert.html type="secondary" content="This is a secondary alert" %}
 
+#### alert.html
 
-### Alerts
-
-{% include alert.html type="info" title="What is an alert?" content="An alert is a box that can stand out to indicate important information. You can choose from levels success, warning, danger, info, and primary. This example is an info box, and the code for another might look like this:" %}
+The [alert.html](https://github.com/meshery/meshery/blob/master/docs/_includes/alert.html) file is used to format and display notes, warnings, and other informational callouts in Meshery. 
```

Comment:
Meshery Docs has a common include file [alert.html](https://github.com/meshery/meshery/blob/master/docs/_includes/alert.html) , to provide consistent formatting for notes, warnings, and various informative callouts intended for the readers.